src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Draw {

```

Comment:
```suggestion
public class Circle extends Figure {

```

if Figure implements AreaCalculator and Draw than all it's children also implements them